src/routes/analytics/new/NewDashboardPage.tsx

```diff
@@ -0,0 +1,39 @@
+import { useAtom } from "jotai"
+import { Flex } from "@mantine/core"
+import {
+  CreateDashboardModal,
+  EditableDashboard,
+} from "@metabase/embedding-sdk-react"
+import { navigate } from "wouter/use-browser-location"
+
+import { RemountOnSiteChange } from "../../../components/RemountOnSiteChange"
+
+import { siteAtom } from "../../../store/site"
+import { createDashboardIdAtom } from "../../../store/create"
+
+import { SANDBOXED_USER_GENERATED_COLLECTIONS } from "../../../constants/collections"
+
+export const NewDashboardPage = () => {
+  const [site] = useAtom(siteAtom)
+  const [dashboardId, setDashboardId] = useAtom(createDashboardIdAtom)
+
+  const collectionId = SANDBOXED_USER_GENERATED_COLLECTIONS[site]
+
+  if (dashboardId === null) {
+    return (
+      <RemountOnSiteChange>
```

Comment:
based on [this comment](https://github.com/metabase/shoppy/pull/60/files?file-filters%5B%5D=.json&file-filters%5B%5D=.lock&file-filters%5B%5D=.ts&file-filters%5B%5D=.tsx&show-viewed-files=true&show-deleted-files=false#r1742553220) that removes `RemountOnSiteChange`, do we still need this here?

Reply:
@WiNloSt yes, this is just to reload the "Which collection should this go in?" dropdown. Otherwise, the collection dropdown still shows the old set of collections.